### ISSUE TITLE
[Failing test] skew-cluster-latest-kubectl-stable1-gce kubetest.Extract

### ISSUE THREAD

**Issue by shuheiktgw** [kubernetes/kubernetes#113762]
### Which jobs are failing?

skew-cluster-latest-kubectl-stable1-gce 

### Which tests are failing?

kubetest.Extract

### Since when has it been failing?

Since 11/8

### Testgrid link

https://testgrid.k8s.io/sig-release-master-blocking#skew-cluster-latest-kubectl-stable1-gce

### Reason for failure (if possible)

```
W1108 16:45:43.298] 2022/11/08 16:45:43 main.go:328: Something went wrong: failed to acquire k8s binaries: Failed to set release from https://storage.googleapis.com/k8s-release-dev/ci/k8s-stable1.txt (Unexpected HTTP status code: 404)
W1108 16:45:43.301] Traceback (most recent call last):
W1108 16:45:43.301]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 723, in <module>
W1108 16:45:43.301]     main(parse_args())
W1108 16:45:43.302]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 569, in main
W1108 16:45:43.302]     mode.start(runner_args)
W1108 16:45:43.302]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 228, in start
W1108 16:45:43.303]     check_env(env, self.command, *args)
W1108 16:45:43.303]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 111, in check_env
W1108 16:45:43.303]     subprocess.check_call(cmd, env=env)
W1108 16:45:43.303]   File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
W1108 16:45:43.304]     raise CalledProcessError(retcode, cmd)
W1108 16:45:43.305] subprocess.CalledProcessError: Command '('kubetest', '--dump=/workspace/_artifacts', '--gcp-service-account=/etc/service-account/service-account.json', '--up', '--down', '--test', '--provider=gce', '--cluster=bootstrap-e2e', '--gcp-network=bootstrap-e2e', '--check-leaked-resources', '--check-version-skew=false', '--extract=ci/k8s-stable1', '--extract=ci/latest', '--extract-ci-bucket=k8s-release-dev', '--gcp-node-image=gci', '--gcp-zone=us-west1-b', '--ginkgo-parallel=25', '--skew', '--test_args=--ginkgo.focus=Kubectl --ginkgo.skip=\\[Serial\\]|\\[Deprecated\\] --minStartupPods=8', '--timeout=120m')' returned non-zero exit status 1
E1108 16:45:43.311] Command failed
I1108 16:45:43.311] process 245 exited with code 1 after 0.1m
E1108 16:45:43.311] FAIL: ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew
```

### Relevant SIG(s)

/sig testing

**Comment by shuheiktgw** [comment]
[Slack thread](https://kubernetes.slack.com/archives/C09QZ4DQB/p1667947284689449)

**Comment by Nivedita-coder** [comment]
cc @kubernetes/ci-signal

**Comment by xmudrii** [comment]
The job will remain red because `ci-kubernetes-build-1.25` requires something to get merged on the release-1.25 branch for the new version maker to get generated. I'll reopen this issue until we don't get the first green run.
/reopen

**Comment by Nivedita-coder** [comment]
job is passing! :)
/close
